tests/test_spatial_tools.py

```diff
@@ -7,8 +7,8 @@
 import numpy as np
 import pytest
 
-import DemUtils as du
-from test_coreg import EXAMPLE_PATHS
+import xdem as du
+# from test_coreg import EXAMPLE_PATHS
```

Comment:
The example paths need to be imported. Otherwise this test won't work.

I guess a better example approach should be made. I think @adehecq mentioned that GeoPandas has a good structure?